adbe/adb_enhanced.py

```diff
@@ -1562,43 +1579,3 @@ def switch_screen(switch_type):
                                  "screen control operation. Error: %s" % e)
 
     return o
-
```

Comment:
Please rebase your code to avoid reverting this addition.

Reply:
Sure. Sorry for that!

Reply:
No worries.